google/cloud/storage/internal/curl_client.cc

```diff
@@ -1079,28 +1080,27 @@ std::pair<Status, ObjectMetadata> CurlClient::InsertObjectMediaMultipart(
   builder.AddQueryParameter("uploadType", "multipart");
   builder.AddQueryParameter("name", request.object_name());
 
-  std::unique_ptr<internal::CurlStreambuf> buf(
-      new internal::CurlStreambuf(builder.BuildUpload(),
-                                  client_options().upload_buffer_size()));
+  std::unique_ptr<internal::CurlStreambuf> buf(new internal::CurlStreambuf(
```

Comment:
Do you think it's worth adding a @note or @see directed toward RFC 1341 (https://tools.ietf.org/html/rfc1341)?  Or to https://cloud.google.com/storage/docs/json_api/v1/how-tos/multipart-upload ?